_laptop/thinkpad_x13s.md

```diff
@@ -0,0 +1,131 @@
+---
+name: ThinkpadX13s
```

Comment:
Having an extra space here for Thinkpad X13s (same for T14) would look better.

Reply:
Added, need to change also the wrapping stuff but done.